erpnext_thailand/public/files/thai_zip_code.csv

```diff
@@ -0,0 +1,7538 @@
+ID,Zip Code,Tambon,Amphur,Province
```

Comment:
I found duplicate some row (same ID, Tambon, Amphur, Province but difference zip code)
![Selection_842](https://github.com/user-attachments/assets/35549b17-30ff-4780-9a99-9f85d221be8e)

I check with this files has 7537 rows but in Thai Zip Code tables has 7436 rows (Number of row should same)